### ISSUE TITLE
Deficiency

### ISSUE THREAD

**Issue by iluadiaz** [eborello/woop#67]
When spelling a movie wrong in the YouTube search bar, it the extension does not work as intended. My guess is the extension looks at the search bar text, and then uses that to blur the movies added to the extension. A screen reader might be needed to look for the key word in the titles or description (description might be too much)
